setup.sh

```diff
@@ -219,6 +219,7 @@ EOF
     echo 'adding the Portal to nginx'
     cat > streetview <<EOF
 server {
+    client_max_body_size 1024M;
```

Comment:
I'm not sure what exactly this restricts, but isn't hard-coding a value like this asking for trouble? If something bumps up against this limit, it might be difficult to figure out what it was and troubleshoot.